src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -288,10 +301,16 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       }
       case 'EvmCustomTx': {
         if (!inboundAddressData?.router) return undefined
+        if (!accountAssetAddress) return undefined
         const assetAddress = isToken(fromAssetId(assetId).assetReference)
           ? getAddress(fromAssetId(assetId).assetReference)
           : zeroAddress
-        const amount = BigInt(toBaseUnit(amountCryptoPrecision, asset.precision).toString())
+        const amount = // Reuse the savers util as a sane amount for the dust threshold
```

Comment:
Here too amount or dust terminology if we want to

Reply:
https://github.com/shapeshift/web/pull/6285/commits/32f59bac73d2f81b3cd1f84ddbbf2c6a3244aa8a